style.css

```diff
@@ -491,4 +492,84 @@ aside {
     cursor: pointer;
     box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
   }
+
+  .footer {
+    position: absolute;
+    background-image: url(Contact\ me.png);
+    top: 599%;
+    padding: 50px 6px 0;
+    width: fit-content;
+    overflow: hidden;
+  }
```

Comment:
kindly fix the padding of your form. please You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).